variables.tf

```
@@ -28,6 +28,12 @@ variable "repository" {
   description = "The name of your infrastructure repo"
 }
 
+variable "runner_image" {
+  type        = string
+  description = "URL of the Docker image used to process Runs. Defaults to `null` which is Spacelift's standard (Alpine) runner image."
+  default     = "null"
```

Shouldn't this be `null` and not `"null"`? I believe passing `null` to an argument always (?) means that it will use the default from the provider, which should be the alpine image like you mentioned. 

We can test this with apply-level tests in this module. Good reason to add those. I can follow up with those sort of thing if you give me the :+1: 

```suggestion
  default     = null
```

---

Ah yeah, supposed to be null, not the string null. Typo.

---

@coderabbitai should've caught this 😢 I wonder if there is a way we can ping it to catch this sort of thing in the future. I'm disappointed @coderabbitai